src/js/helpers/analytics/Analytics.js

```diff
@@ -31,30 +31,27 @@ const Analytics = {
         if (!args.category || !args.action) {
             return;
         }
-        if (kGlobalConstants.QAT) {
+        if (args.gtm) {
             window.dataLayer = window.dataLayer || [];
             window.dataLayer.push({
-                event: 'event',
-                eventProps: {
-                    category: `${this._prefix}${args.category}`,
-                    action: args.action,
-                    label: args.label || undefined,
-                    value: args.value || undefined,
-                    nonInteraction: args.nonInteraction || undefined
-                }
+                event: args.event || 'event',
+                event_category: `${this._prefix}${args.category}`,
+                event_action: args.action,
+                event_label: args.label || undefined,
+                event_value: args.value || undefined,
+                event_nonInteraction: args.nonInteraction || undefined
```

Comment:
does this need an event_gtm: somevalue

Reply:
no, for this condition i'm just checking if gtm is true then create the google tag manager event.  the previous team had started adding exploring gtm so this code was partially there already

`        if (args.gtm) {
            window.dataLayer = window.dataLayer || [];
            window.dataLayer.push({
                event: args.event || 'event',
                event_category: `${this._prefix}${args.category}`,
                event_action: args.action,
                event_label: args.label || undefined,
                event_value: args.value || undefined,
                event_nonInteraction: args.nonInteraction || undefined`